### PR TITLE
Fixed missing address street field

### PR DESCRIPTION
**Pull request by BLasan**
## Description
See #3343 

## Related issues and discussion
#3343 

## Screenshots, if any
Screenshot from 2021-01-28 01-43-15

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [ ] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by BLasan** [comment]
@bharathcgowda @spider0061  Please review

**Comment by bharathcgowda** [comment]
@spider0061  Hi Ashutosh, would you please help us in reviewing and merging this?

**Comment by spider0061** [comment]
> @spider0061 Hi Ashutosh, would you please help us in reviewing and merging this?

Sure Sir.

**Review by spider0061 — APPROVED** [review]
LGTM!

**Comment by bharathcgowda** [comment]
@edcable  can we get this merged by someone, or could I merge it as it is already being approved by @spider0061

File: app/scripts/controllers/client/CreateClientController.js [hunk 1]
```diff
@@ -37,6 +37,8 @@
             scope.tf = "HH:mm";
             scope.clientId = routeParams.clientId;
```
Comment (wkk91193) [inline app/scripts/controllers/client/CreateClientController.js:39]: @BLasan Please remove the console log statements

**Review by wkk91193 — CHANGES_REQUESTED** [review]
@BLasan Please remove the console log statements

**Comment by BLasan** [comment]
> @BLasan Please remove the console log statements

Done. Please review

**Pull request merged**